### PR TITLE
Add automation, simplify README & make targets

### PR DESCRIPTION
**Pull request by robertlcx**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,184 +42,128 @@ This solution has the EKS cluster deployed on AWS.
...
 
-Test the deploy AWS CDK solution.
+### Multiple Namespaces
 
-```shell
-make integ-awscdk-test
-```
+To deploy multiple Localstack instances with their own dev environment, you can do something like this:
 
-#### Setup Second Namespace
```
Comment (cabeaulac) [inline README.md:193]: Keep this section in the readme for easy reference to basic cmd line steps

**Review by cabeaulac — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (robertlcx) [reply README.md:193]: We added a `### Multiple Namespaces` section that creates `n` number of these environments. Wouldn't that be more comprehensive though? What do you say?

Comment (robertlcx) [reply README.md:193]: On second note: we still want this for strictly the demo.

Comment (robertlcx) [reply README.md:193]: Done!

**Review by cabeaulac — APPROVED** [review]
LGTM

**Pull request merged**